packages/block-library/src/navigation/editor.scss

```
@@ -158,7 +157,7 @@ $colors-selector-size: 22px;
 		min-width: $colors-selector-size;
 		height: $colors-selector-size;
 		min-height: $colors-selector-size;
-		line-height: ( $colors-selector-size - 2 );
+		line-height: ($colors-selector-size - 2);
```

```suggestion
		line-height: ( $colors-selector-size - 2 );
```

This change seems unrelated.